cdc/model/sink.go

```diff
@@ -270,6 +270,8 @@ type RowChangedEvent struct {
 
 	// SplitTxn marks this RowChangedEvent as the first line of a new txn.
 	SplitTxn bool `json:"-" msg:"-"`
+	// Filtered marks this RowChangedEvent was filtered by filter in mounter.
+	Filtered bool `json:"-" msg:"-"`
```

Comment:
Why do we need to expose it? It does not make sense to expose it for sink.

Reply:
done PTAL